p2pool/src/sharechain/p2chain.rs

```diff
@@ -233,10 +232,7 @@ impl<T: BlockCache> P2Chain<T> {
                 warn!(target: LOG_TARGET, "Block squad mismatch, skipping block");
                 continue;
             }
-            debug!(target: LOG_TARGET, "Loading block {}({:x}{:x}{:x}{:x}) into chain", block.height, block.hash[0], block.hash[1], block.hash[2], block.hash[3]);
-            if i % 250 == 0 {
-                info!(target: LOG_TARGET, "Loading block {} into chain", i);
-            }
+            info!(target: LOG_TARGET, "Loading block {}({:x}{:x}{:x}{:x}) into chain", block.height, block.hash[0], block.hash[1], block.hash[2], block.hash[3]);
```

Comment:
done change this log here, keep the original pattern of only doing every %250 to info